test/odbc/constants.cpp

```diff
@@ -1,6 +1,11 @@
 #include "constants.h"
 using std::string;
 
+map<constants::ServerType, string> constants::server_to_odbc_types = { 
+    {ServerType::MSSQL, "MSSQL"}, 
+    {ServerType::PSQL, "PSQL"} 
```

Comment:
Hard coding these values will require code change to add additional driver.
It is probably not necessary even for current functionality. 

Both servertype and the corresponding mapping can be specified in the configuration file and this map (if it is needed) can be dynamically created based on the configuration file or the corresponding command line